playground/tree-sitter/build.py

```diff
@@ -138,9 +183,13 @@ def copy_wasm_to_assets(wasm_file: Path, lang_name: str, assets_dir: Path) -> Pa
 def instantiate_index_html(template_path: Path, output_path: Path):
     with template_path.open("r") as inp, output_path.open("w") as out:
         content = inp.read()
-        languages = [
-            f'<option value="{lang}">{lang.title()}</option>' for lang in _LANGUAGES
-        ]
+        languages = []
+        for lang in _LANGUAGES:
+            # Set python as the default language
+            if lang == "python":
+                languages.append(f'<option value="{lang}" selected>{lang.title()}</option>')
```

Comment:
hmmm, not opposing this, but why? IIRC we don't even support python in the playground